.gitignore

```diff
@@ -46,3 +46,4 @@ bin/
 lib/
 lib64
 pyvenv.cfg
+snap/snapd-desktop-integration/current
```

Comment:
Could you please remove this change. This is part of your setup and not something that should be included into LibreNMS repo.